src/main/java/co/cask/wrangler/internal/TextSpecification.java

```diff
@@ -66,8 +69,33 @@
   // directives for wrangling.
   private String[] directives;
 
+  // Mapping of specification formats.
+  Map<String, String> formats = new HashMap<>();
+
   public TextSpecification(String[] directives) {
     this.directives = directives;
+
+    // Add all the usages.
+    formats.put("set format", "set format [csv|json] <delimiter> <skip empty lines>");
```

Comment:
@russorat don't have a great idea right now on how to make it easy. Can I leave it for now and address it little latter ?